db/migrate/20151130214422_create_movies.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMovies < ActiveRecord::Migration
+  def change
+    create_table :movies do |t|
+      t.string :name
+      t.string :director
+      t.string :description
+      t.integer :upvotes, :default => 0
```

Comment:
Nice way to ensure that there is a starting value here so that you don't have to deal with it when you create your initial Movie records.